frontend/src/components/Graph.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Plot from 'react-plotly.js';
+
+const Graph=({accounts})=>{
+    const vals = []
+    const labels = []
+    accounts.map((acct) => 
```

Comment:
`map` is generally used when you want to use the return value and `forEach` is for iterating.  In this case, it looks like you want the return value, but you aren't using it.  Instead, it might be valuable to do:

```js
const values = accounts.map((account) => account.amount)
const labels = accounts.map((account) => acct.name)
```